src/main/java/org/intellij/sdk/codesync/activityAlert/ActivityAlertNotification.java

```diff
@@ -0,0 +1,31 @@
+package org.intellij.sdk.codesync.activityAlert;
+
+import com.intellij.diagnostic.ReportMessages;
+import com.intellij.notification.NotificationListener;
+import com.intellij.notification.NotificationType;
+import com.intellij.openapi.project.Project;
+import org.intellij.sdk.codesync.Constants;
+
+import java.util.ArrayList;
+
+public class ActivityAlertNotification {
+
+    public static void showAlert(String teamActivityURL, boolean isTeamAcitivty, Project project, String email){
+
+        ArrayList<AlertAction> actions = new ArrayList<>();
+        actions.add(new AlertAction("View Activity", project, email, teamActivityURL));
+        actions.add(new AlertAction("Remind Later", project, email, teamActivityURL));
+        actions.add(new AlertAction("Skip Today", project, email, teamActivityURL));
+
+        String NotificationMessage = isTeamAcitivty? Constants.Notification.TEAM_ACTIVITY_ALERT_MESSAGE: Constants.Notification.ACTIVITY_ALERT_MESSAGE;
+
+        ReportMessages.GROUP.createNotification(
```

Comment:
Getting a warning at GROUP as 
<img width="865" alt="image" src="https://github.com/codesyncapp/codesync-intellij/assets/9863016/6ebfa850-1cd2-49b7-8732-b49a94125015">

Reply:
Are we not fixing this right now @syedgulahmed?

Reply:
Working on this. Will update other place where it is being used otherwise will keep it same here too.